Laboratory Work #3

ArtemSultanov opened the pull request.

ArtemSultanov committed:
Create README.md

ArtemSultanov committed:
Create OWASP-Top10-VerComp.md

ArtemSultanov committed:
Create OWASP-Top10-Software-Analysis.md

oleksandrblazhko commented:
Рішення розглянемо під час спілкувань на консультації або на занятті.

oleksandrblazhko commented:
1) у другому завданні цієї роботи у вас є опис "A2 – Cryptographic Failures"
У роботі № 1 є опис FR, для яких є загроза порушення конфіденційності 
https://github.com/OS-IS/ai201-sultanov/tree/Laboratory-Work-3/Laboratory-work-1/1-FR-NFR
Конфіденційність забезпечується криптографічними алгоритмами.
Тому треба оновити перелік FR, для яких є така загроза.

ArtemSultanov committed:
update OWASP-Top10-Software-Analysis.md

Перелік FR було оновлено згідно з аналізом загроз конфіденційності у попередній лабораторній роботі, зокрема, додано FR1, FR2, FR5, FR6 для A2 – Cryptographic Failures.

oleksandrblazhko commented:
Оцінка = 5 балів
